tools/gce-quota-sync/gce-quota-sync.py

```diff
@@ -132,15 +132,15 @@ def _quota_to_series(project, region, quota):
     region: set in converted time series labels
     quota: quota object received from the GCE API
   """
-  labels = dict((k, str(v)) for k, v in quota.items() if k != 'usage')
+  labels = dict((k, str(v)) for k, v in quota.items())
+
   labels['project'] = project
   labels['region'] = region
   try:
-    value = quota['usage'] / quota['limit']
+    value = quota['usage'] / float(quota['limit'])
```

Comment:
IIUC python3 this [casting as float is unecessary](https://www.python.org/dev/peps/pep-0238/) when using division operator.

Reply:
Exactly :) My client ran it on python2, figured it wouldn't hurt.

Reply:
This is fine for backwards compatibility but your client should get w/ the times :)

Reply:
Haha yes.